modules/navier_stokes/include/userobjects/INSFVRhieChowInterpolator.h

```diff
@@ -59,21 +46,21 @@ class INSFVRhieChowInterpolator : public GeneralUserObject,
    * @param component The velocity component we are adding 'a' coefficient data for
    * @param value The value of 'a' that we are adding
    */
-  void addToA(const libMesh::Elem * elem, unsigned int component, const ADReal & value);
+  void addToA(const libMesh::Elem * elem, unsigned int component, const ADReal & value) override;
 
   /**
    * Retrieve a face velocity
-   * @param m The velocity interpolation method. This is either Rhie-Chow or Average. Rhie-Chow is
-   * recommended as it avoids checkerboards in the pressure field
    * @param fi The face that we wish to retrieve the velocity for
    * @param time The time at which to evaluate the velocity
-   * @param tid The thread ID
+   * @param tid The thread ID\
+   * @param m The velocity interpolation method. This is either Rhie-Chow or Average. Rhie-Chow is
+   * recommended as it avoids checkerboards in the pressure field
    * @return The face velocity
    */
-  VectorValue<ADReal> getVelocity(Moose::FV::InterpMethod m,
```

Comment:
We’re just re ordering the arguments here? If so seems like an unnecessary diff?

Reply:
I reordered it because for segregated solvers doesn't make sense to pass the interpolation method. I will try to figure out something nicer here!

Reply:
I reset the original ordering.